graphrag/config/create_graphrag_config.py

```diff
@@ -318,13 +318,16 @@ def hydrate_parallelization_params(
             reader.envvar_prefix(Section.node2vec),
             reader.use(values.get("embed_graph")),
         ):
+            use_lcc = reader.bool("use_lcc")
```

Comment:
Wonder if we should start just defaulting these and removing them from config. (Explicitly defaulting in this section)

Reply:
Some of these I have been removing, but this one was kind of weird. It seems perfectly reasonable to allow users to specify whether to only use the lcc, but to my knowledge we always set this to True. Maybe a good question for Ha

Reply:
I'm not sure if any users actually change this value, but for lazygraphrag I actually set it to False by default (so that we don't lose nodes).

Reply:
Ok, I will leave the config in place for now given a False use case